fdbclient/BlobCipher.cpp

```diff
@@ -49,6 +57,152 @@
 #endif
 
 #define BLOB_CIPHER_DEBUG false
+#define BLOB_CIPHER_SERIALIZATION_CHECKS false
+
+// BlobCipherEncryptHeaderRef
+
+uint32_t BlobCipherEncryptHeaderRef::getHeaderSize(const int flagVersion,
+                                                   const int authAlgoVersion,
+                                                   const EncryptCipherMode cipherMode,
+                                                   const EncryptAuthTokenMode authMode,
+                                                   const EncryptAuthTokenAlgo authAlgo) {
+	uint32_t total = sizeof(uint16_t) * 2; // sizeof(flagVersion + algoHeaderVersion)
+	if (flagVersion != 1) {
+		throw not_implemented();
+	}
+
+	total += sizeof(BlobCipherEncryptHeaderFlagsV1);
+
+	if (cipherMode == ENCRYPT_CIPHER_MODE_AES_256_CTR) {
+		throw not_implemented();
```

Comment:
isn't it the only available mode right now?

Reply:
seems like fallout from my minor changes before PR publish, shall fix it.